Dockerfile

```diff
@@ -17,8 +17,8 @@
 # under the License.
 #
 
-ARG GO_VERSION=golang:1.13
-FROM apachepulsar/pulsar:latest as pulsar
+ARG GO_VERSION=golang:1.15
+FROM apachepulsar/pulsar:2.8.1 as pulsar
```

Comment:
Please do not make this modification, we may need to check why the latest docker image cannot pass Action CI, and then create a corresponding issue to fix this problem.

Reply:
I already created the issue, https://github.com/apache/pulsar-client-go/issues/748. But, without the modification, let the publish of pulsar broker blocking all the pr ?